client/blocks/plugins-scheduled-updates/notification-settings.scss

```diff
@@ -0,0 +1,24 @@
+@import "@automattic/color-studio/dist/color-variables";
+@import "@automattic/typography/styles/variables";
+
+.plugins-update-manager {
+	.notification-settings-form {
+		label {
+			font-size: $font-body-small;
+			font-weight: 500;
+			color: $studio-black;
+			margin-bottom: 8px;
```

Comment:
Throughout the module, we rely on the `rem` unit for the margin and padding properties.
Please update it accordingly.

`margin-bottom: 0.5rem`

Reply:
Thanks for the pointer, I'll update it 👍